src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -104,6 +104,27 @@ public static Deadline parseDeadline(String deadline, String startDate) throws P
         return new Deadline(trimmedDeadline, trimmedStartDate);
     }
 
+    /**
+     * Parses a {@code String deadline} into an {@code Deadline}.
+     * This is only for modify command, since user might not include startDate in the command
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code deadline} is invalid.
+     */
+    public static Deadline parseDeadline(String deadline) throws ParseException {
+        requireNonNull(deadline);
+        String trimmedDeadline = deadline.trim();
+        if (!Deadline.isValidDate(trimmedDeadline)) {
+            throw new ParseException(Deadline.MESSAGE_CONSTRAINTS);
+        }
+        String year = trimmedDeadline.substring(trimmedDeadline.length() - 4);
+        String placeHolderStartDate = "31/12/" + year;
```

Comment:
perhaps can use LocalDate.max instead of hardcoding

Reply:
thanks for the suggestion